compute/cloud-client/instances/src/list_all_instances.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/compute/cloud-client/README.md
+ */
+
+namespace Google\Cloud\Samples\Compute;
+
+# [START compute_instances_list_all]
+use Google\Cloud\Compute\V1\InstancesClient;
+
+/**
+ * List all instances for particular $projectId
+ * Example:
+ * ```
+ * list_all_instances($projectId);
+ * ```
+ *
+ * @param string $projectId Your Google Cloud project ID.
+ *
+ * @throws \Google\ApiCore\ApiException if the remote call fails.
+ */
+function list_all_instances(string $projectId)
+{
+    // List the new Compute Engine instance using the InstancesClient
+    $instancesClient = new InstancesClient();
+    $aggList = $instancesClient->aggregatedList($projectId);
```

Comment:
Let's just call this variable `$allInstances`, I think that name is more descriptive of what it actually is
```suggestion
    $allInstances = $instancesClient->aggregatedList($projectId);
```